GitUI/HelperDialogs/FormProcess.cs

```diff
@@ -23,12 +22,13 @@ public class FormProcess : FormStatus
         public HandleOnExit HandleOnExitCallback { get; set; }
         public readonly Dictionary<string, string> ProcessEnvVariables = new Dictionary<string, string>();
 
-        protected FormProcess()
-            : base(true)
+        [Obsolete("For VS designer and translation test only. Do not remove.")]
+        public FormProcess()
```

Comment:
Why public? Reflection does not need it.

Reply:
reverted